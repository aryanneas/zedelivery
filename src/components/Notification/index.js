import React from 'react'
import { ToastContainer, toast, Slide } from 'react-toastify'
import 'react-toastify/dist/ReactToastify.css'

const Notification = () => {
  return (
    <ToastContainer
      transition={Slide}
      position="top-center"
      closeButton={false}
      bodyClassName="ToastBody"
    />
  )
}

Notification.emitError = toast.error
Notification.emitSuccess = toast.success
Notification.emitInfo = toast.info

export default Notification
