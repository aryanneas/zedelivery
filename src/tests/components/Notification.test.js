import React from 'react'
import { render } from '@testing-library/react'

import Notification from '../../components/Notification'

describe('Notification component', () => {
  it('should render and consume notifications', async () => {
    const { asFragment } = render(<Notification />)
    expect(asFragment()).toMatchSnapshot()
  })
})
