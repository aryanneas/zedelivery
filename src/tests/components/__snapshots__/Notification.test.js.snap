// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification component should render and consume notifications 1`] = `
<DocumentFragment>
  <div
    class="Toastify"
  />
</DocumentFragment>
`;
